app/javascript/react/src/components/Greeting.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useState } from 'react';
+import fetchData from "../redux/welcome/fetchAPi";
+
+const RandomMessage = () => {
+  const [msg, setMsg] = useState('');
+  const state = useSelector((state) => state.Random);
+  const dispatch = useDispatch();
+  
+  useEffect(() => {
+    if (state.data && state.data.message) {
+      setMsg(state.data.message);
+    } else {
+      dispatch(fetchData());
+    }
+  }, [dispatch, state.data]);
+
```

Comment:
- [ ] I'm not able to see the greetings on the display, It's showing a blank screen. The back-end is working fine but I think there are some issues with the front-end configurations. Please make sure to check, fix it and random greetings should display on the screen.

![image](https://github.com/BHObserver/hello-rails-react/assets/83803414/c1ad12af-71e2-4945-92f2-d33e98c17d18)


![image](https://github.com/BHObserver/hello-rails-react/assets/83803414/a38de2e7-b851-435a-ada8-64c9d40b0554)